buildscripts/cstor-repl-ctrl/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#
+# This Dockerfile builds a recent maya nodebot using the latest binary from
+# cstor-repl-ctrl  releases.
+#
+
+FROM openebs/cstor-base:ci
```

Comment:
Add a comment describing what is contained in the openebs/cstor-base:ci and why are you using a "ci" tag? can it be versioned?

Reply:
will add a comment regarding the binaries it contains. This image is built and pushed from travis when the commit is made onto openebs/cstor. Since no release is made from openebs/cstor, we have got ci tag, when we make release from github, we'll get the required TRAVIS_TAG or latest tag. As of now, we'll continue to use ci tag to pass unit test cases which is planned to be written soon. Please confirm

Reply:
sure. can you put in a issue to track this for 0.7 release.

Reply:
done